### DISCUSSION TITLE
Tools/AWS/Compute/ECS/ECS

### DISCUSSION THREAD

**Discussion by giscus** [xy-241/CS-Notes#4] [General]
# Tools/AWS/Compute/ECS/ECS

Abstract § A lightweight container orchestration service We can mount Volume to it for data that can be lost when Container is terminated Components § Container § We can obtain a shell into it using ECS Exec Obtain Private IP of the container, must have curl &amp;&amp; jq (JSON Processor) containerPrivateIP=$(curl $ECS_CONTAINER_METADATA_URI | jq -r &#039;.

https://notes.yxy.ninja/Tools/AWS/Compute/ECS/ECS

**Comment by OliverTeo288** [comment]
Good job XY !

**Reply by xy-241** [reply]
![Happy NSF](https://i.gifer.com/origin/65/651d5042e49e2eea63ea94c09e119716_w200.gif)
 Thank you Sir!
